### ISSUE TITLE
Print bootstrap and jre (both class and js) after annotations.

### ISSUE THREAD

**Issue by mP1** [mP1/j2cl-maven-plugin#304]

**Comment by mP1** [comment]
```txt
[DEBUG]   Metadata
[DEBUG]     0 Annotation processor
[DEBUG]     2 Annotation only class files
[DEBUG]       com.google.jsinterop:jsinterop-annotations:jar:2.0.0
[DEBUG]       walkingkooka:j2cl-gwt-internal-annotations:pom:1.0-SNAPSHOT
[DEBUG]     8 Classpath required
[DEBUG]       com.google.jsinterop:jsinterop-annotations:jar:2.0.0
[DEBUG]       walkingkooka:j2cl-bootstrap-javascript:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-gwt-internal-annotations:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-javac-bootstrap-classpath:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-jre-java:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-jre-javascript:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-uber:jar:1.0-SNAPSHOT
[DEBUG]       walkingkooka:jsinterop-base:pom:1.0-SNAPSHOT
[DEBUG]     1 JRE bootstrap class files
[DEBUG]       walkingkooka:j2cl-javac-bootstrap-classpath:pom:1.0-SNAPSHOT
[DEBUG]     1 JRE class files
[DEBUG]       walkingkooka:j2cl-jre-java:pom:1.0-SNAPSHOT
[DEBUG]     6 Ignored dependencies
[DEBUG]       com.google.jsinterop:jsinterop-annotations:jar:2.0.0
[DEBUG]       walkingkooka:j2cl-bootstrap-javascript:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-gwt-internal-annotations:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-javac-bootstrap-classpath:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-jre-java:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-jre-javascript:pom:1.0-SNAPSHOT
[DEBUG]     6 Javascript source required
[DEBUG]       com.google.jsinterop:jsinterop-annotations:jar:2.0.0
[DEBUG]       walkingkooka:j2cl-bootstrap-javascript:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-gwt-internal-annotations:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-jre-javascript:pom:1.0-SNAPSHOT
[DEBUG]       walkingkooka:j2cl-uber:jar:1.0-SNAPSHOT
[DEBUG]       walkingkooka:jsinterop-base:pom:1.0-SNAPSHOT
[DEBUG]     1 Javascript bootstrap class files
[DEBUG]       walkingkooka:j2cl-bootstrap-javascript:pom:1.0-SNAPSHOT
[DEBUG]     1 Javascript class files
[DEBUG]       walkingkooka:j2cl-jre-javascript:pom:1.0-SNAPSHOT
```

**Comment by mP1** [comment]
- Closed by #307
